helix-core/src/main/java/org/apache/helix/tools/ClusterSetup.java

```diff
@@ -189,6 +189,16 @@ public void addCluster(String clusterName, boolean overwritePrevious, CloudConfi
 
     if (cloudConfig != null) {
       _admin.addCloudConfig(clusterName, cloudConfig);
+      // If cloud is enabled and Cloud Provider is Azure, populated the Topology information in cluster config
+      if (cloudConfig.isCloudEnabled()
+          && cloudConfig.getCloudProvider().equals(CloudProvider.AZURE.name())) {
+        ConfigAccessor configAccessor = new ConfigAccessor(_zkServerAddress);
+        ClusterConfig clusterConfig = new ClusterConfig(clusterName);
+        clusterConfig.setTopology("/helixZoneId/host");
```

Comment:
Are we agree use this? I thought i was "/faultDomain/hostname"? Anyway, we should put it into AzureConstant.

Reply:
Done.